components/migrate/landing/Index.vue

```diff
@@ -2,6 +2,48 @@
   <div class="mt-8">
     <MigrateLandingHeader />
     <hr />
-    <MigrateLandingContent />
+    <div class="flex flex-col md:flex-row gap-4 justify-between">
+      <div>
+        <div
+          v-if="accountId"
+          class="flex flex-col md:flex-row gap-4 items-center">
+          <p class="mr-4">{{ $t('migrate.resultFor') }}</p>
+          <div class="flex">
+            <Avatar :value="accountId" :size="26" class="mr-2" />
+            <nuxt-link class="has-text-k-blue" :to="`/ksm/u/${accountId}`">
```

Comment:
`has-text-k-blue` any tailwind equivalent maybe `text-text-k-blue`

force link to `/ksm/u/`? no prefix?